flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/KafkaTopicsDescriptor.java

```diff
@@ -51,8 +50,19 @@ public KafkaTopicsDescriptor(
             checkArgument(
                     !fixedTopics.isEmpty(),
                     "If subscribing to a fixed topics list, the supplied list cannot be empty.");
+            fixedTopics.forEach(
+                    topic -> {
+                        checkArgument(
+                                !org.apache.flink.util.StringUtils.isNullOrWhitespaceOnly(topic),
```

Comment:
Sorry for the misleading, we don't need to specific org.apache.flink.util.StringUtils.isNullOrWhitespaceOnly, we can just import org.apache.flink.util.StringUtils and use  StringUtils.isNullOrWhitespaceOnly here.

The reason we need to specific org.apache.flink.util.StringUtils.isNullOrWhitespaceOnly in class FlinkKafkaProducer.java is that we have imported org.apache.commons.lang3.StringUtils.